src/Compiler/Checking/import.fs

```diff
@@ -108,7 +108,9 @@ let createTypeSubsumptionCache (g: TcGlobals) =
                 MaximumCapacity = 4 * 32768 }
     Cache.Create<TTypeCacheKey, bool>(options)
 
-let typeSubsumptionCaches = ConditionalWeakTable<TcGlobals, Cache<TTypeCacheKey, bool>>()
+let typeSubsumptionCaches = Cache.Create<TcGlobals, Cache<TTypeCacheKey, bool>>({ CacheOptions.Default with MaximumCapacity = 16 })
```

Comment:
Each cache held here roughly corresponds to one assembly being built / one incremental builder. My understanding is that there is not much turnover here, unless something on the project level changes.
In a test run the story is different, hundreds can be created and disposed.